jax/experimental/jax2tf/README.md

```diff
@@ -386,16 +394,86 @@ jax2tf.convert(lambda x: jnp.sum(x, axis=0) / x.shape[0],
                polymorphic_shapes=["(v, _)"])(np.ones((4, 4)))
 ```
 
-## Caveats
+## Known issues
 
 ### Incomplete TensorFlow data type coverage
 
 There are a number of cases when the TensorFlow ops that are used by the
-jax2tf converter are not supported by TensorFlow for fewer data types than JAX.
+jax2tf converter are not supported by TensorFlow for the same data types as in JAX.
 There is an
 [up-to-date list of unimplemented cases](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/g3doc/primitives_with_limited_support.md).
 
-### Missing features
+There are two kinds of errors you may see. For the primitives in the
+[unimplemented cases](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/g3doc/primitives_with_limited_support.md)
+that are shown to be undefined on all devices and for all execution modes
+(`eager`, `graph`, `compiled`), e.g., `lax.min` for booleans,
+the conversion typically uses a TensorFlow operator that is not
+registered for a certain data type:
+
+```python
+jax2tf.convert(lambda x: lax.min(x, x))(np.array([True]))
+
+>>> InvalidArgumentError: Value for attr 'T' of bool is not in the list of allowed values:
+>>>    bfloat16, half, float, double, uint8, int16, int32, int64;
+>>>    NodeDef: {{node Minimum}};
+>>>    Op<name=Minimum; signature=x:T, y:T -> z:T; attr=T:type,allowed=[DT_BFLOAT16, DT_HALF, DT_FLOAT, DT_DOUBLE, DT_UINT8, DT_INT16, DT_INT32, DT_INT64]> [Op:Minimum]
+```
+
+In the above cases, you should file a bug with JAX or TensorFlow, or consider
+changing your JAX code. We are working on eliminating this kind of problem.
+
+In other cases, the TensorFlow op is registered for the data type, but for the
+`eager` or `graph` execution modes there is no TensorFlow kernel defined.
+Such primitives appear in the
+[unimplemented cases](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/g3doc/primitives_with_limited_support.md)
+as unimplemented for `eager` and `graph`, e.g., `lax.sign` for unsigned integers:
+
+```python
+jax2tf.convert(lax.sign)(np.array([5], dtype=np.uint32))
+
+>>> NotFoundError: Could not find device for node: {{node Minimum}} = Acos[T=DT_UINT32]
+>>> All kernels registered for op Minimum:
+>>>  device='CPU'; T in [DT_FLOAT]
+>>>  device='CPU'; T in [DT_DOUBLE]
+>>>  ...
+```
+
+In this situation, you can still run the converted program if you compile it with
+XLA:
+```python
+tf.function(jax2tf.convert(lax.sign),
+            autograph=True, jit_compile=True)(np.array([5], dtype=np.uint32))
```

Comment:
Here, you have `autograph=True`, whereas before it was set to false. Could you describe (for my own learning) the nuances here?

Reply:
This is a mistake. It never makes sense to set `autograph=True`, because it would ask autograph to make sense of the jax2tf converter. It occasionally gives warnings, and in very rare occasions it fails (this happened last a while ago).